docs/content/docs/dev/table/sql/jar.md

```diff
@@ -51,10 +48,15 @@ The following examples show how to run `JAR` statements in [SQL CLI]({{< ref "do
 {{< tab "SQL CLI" >}}
 ```sql
 Flink SQL> ADD JAR '/path/hello.jar';
-[INFO] The specified jar is added into session classloader.
+[INFO] Execute statement succeed.
+
+Flink SQL> ADD JAR 'hdfs:///udf/common-udf.jar';
+[INFO] Execute statement succeed.
 
 Flink SQL> SHOW JARS;
+jars
 /path/hello.jar
+hdfs:///udf/common-udf.jar
```

Comment:
Is this the style printed by the SQL CLI? Is there a table format around the result?

Reply:
Yes, you are right